examples/research_projects/codeparrot/scripts/arguments.py

```diff
@@ -26,6 +26,9 @@ class TrainingArguments:
     shuffle_buffer: Optional[int] = field(
         default=1000, metadata={"help": "Size of buffer used to shuffle streaming dataset."}
     )
+    batch_shuffle_buffer: Optional[int] = field(
+        default=100000, metadata={"help": "Size of buffer used to shuffle batch sequences of a streaming dataset."}
+    )
```

Comment:
maybe we could share the shuffle buffer variable between streaming and batching. what do you think about 10k as a compromise?